cdap-validation/bin/test_helpers.py

```diff
@@ -0,0 +1,84 @@
+import os
+import base64
+import urllib2
+import json
+import sys
+
+def usage():
+    print "\nThis is the usage function\n"
+    print 'Usage: \n'+sys.argv[0]+' [-v, --verbose] -c, --cluster \'<cluster>\' [-m, --modules \'<modules>]\' -u --user <user:password> -U --uri <URI>\n'
+
+    print """
+    -h, --help  help menu
+    -c, --cluster       cluster name used in API calls to the install manager
+    -m, --modules       run modules, e.g. --modules 'module1;module2'
+    -u, --user          authentication, e.g. --user 'george:pass'
```

Comment:
use multiple args instead of clubbing into one. Less can go wrong in terms of parsing, etc.